README.md

```diff
@@ -62,11 +62,12 @@ To create a `MailingList` you can follow this example. Please, refer to the [Mai
 use Fazland\MailUpRestClient\MailingList;
 
 $email = "owner_of_the_list@email.com";
+$name = "owner_name";
```

Comment:
It's not the owner name but the list name :)
Just a little clarification for the docs.

Reply:
Ok looks good, thank you

Reply:
Can you adjust the code?